spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { build(:post) }
+
+  it "is valid with valid attributes" do
```

Comment:
Let's have a `description` block for "validation" and then inside are two `context` blocks for a post with body and one without body. 

Then we should test associations.

The scopes are a bit complicated to test (see my comment in `post.rb`)